warplib/src/maths/utils.cairo

```diff
@@ -1,72 +1,10 @@
-from starkware.cairo.common.math import split_felt, unsigned_div_rem
-from starkware.cairo.common.math_cmp import is_le
-from starkware.cairo.common.uint256 import Uint256, uint256_le
+use integer::u256_from_felt252;
+use integer::u128_to_felt252;
 
-func get_max{range_check_ptr}(op1, op2) -> (result: felt) {
-    let le = is_le(op1, op2);
-    if (le == 1) {
-        return (op2,);
-    } else {
-        return (op1,);
+fn u256_to_felt252(x: u256) -> felt252 {
```

Comment:
Let's keep the convention of corelib `try_u256_to_felt252`

Reply:
Also shouldn't it be in integer.cairo? We've got a file dealing with integers already

Reply:
> Also shouldn't it be in integer.cairo? We've got a file dealing with integers already

Yes it's a better location, I was respecting the status quo where this kind of function was located in

>  Let's keep the convention of corelib felt252_try_from_u256

You can see both types in corelib, a try and a not try. e.g: `u32_to_felt252`

Reply:
> [#1045 (comment)](https://github.com/NethermindEth/warp/pull/1045#discussion_r1208123402)
> 
> I've made a search for functions that start with "try_" in the Cairo codebase and in all cases it's functions that might return a failure (`Option`), but not panic. Since this function may panic, it might probably make sense to prefix it with "unsafe".
> 
> cc @piwonskp @rodrigo-pino
> 
> (for some reason, I can't reply in the thread itself)

I see, you're right. There's barely any function using panic in the corelib. The convention is established for functions returning Option